src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -67,6 +67,7 @@ class Response
     const HTTP_PRECONDITION_REQUIRED = 428;                                       // RFC6585
     const HTTP_TOO_MANY_REQUESTS = 429;                                           // RFC6585
     const HTTP_REQUEST_HEADER_FIELDS_TOO_LARGE = 431;                             // RFC6585
+    const HTTP_UNAVAILABLE_FOR_LEGAL_REASONS = 451;
```

Comment:
@stof It hasn't been assigned one yet.

As I understand it's been accepted but the RFC editor has some things still to do before it's published and assigned a number.